### DISCUSSION TITLE
Suggestion: Warehouse Hub

### DISCUSSION THREAD

**Discussion by IDragonfire** [voxeltycoon/issues#394] [General] [answered]
## Problem
In the late game you have a lot of truck and train stations in your city to fulfill the demand.

## Idea
Research a warehouse hub (with drones, bicycle or trucks).
Hubs can receive goods from warehouse and sends drones, bicycle or trucks to multiple companies.
This special vehicle distribute multiple goods to companies.

## Improvements
You don't need for every company a station.

## New challange
You deliver directly to the company and must keep an eye on the traffic in your city (street and air)

## Alternative
* Allow trucks to deliver companies directly, see https://github.com/voxeltycoon/suggestions/discussions/317
* add an upgrade/new building to the company (without require more size) to deliver directly

## Summary
You must still deliver the good from warehouse to store, but you need no station for the company.

Source: https://steamcommunity.com/app/732050/discussions/0/3133918821788710380/

**Comment by katepolak** [comment, marked as answer]
For the most part this is a duplicate of #475 which already contains a few ideas for how this could work.
